### DISCUSSION TITLE
Just installed, config.yaml will not validate

### DISCUSSION THREAD

**Discussion by jmayes2** [sbidy/wiz_light#95] [Q&A]
Just installed as per git-hub instructions (Can't find this in HACS).

Reboots fine but after I add the bulb to the yaml and try to validate it all I get is a spinning wheel, it never says my config is good or bad.  If I remove the bulb the yaml checks fine.  I am afraid to reboot without a green light on the yaml.

Please advise,
TX,
J

**Comment by ChrisLizon** [comment]
I'm not sure if home assistant will get hung up if the custom Component hasn't been loaded. Have you restarted home assistant since you extracted the Wiz_Light files?

I would suggest remove the bulb, validate the config, restart home assistant, re-add the bulb, re-validate, and hopefully it gets picked up at that point.

**Reply by jmayes2** [reply]
It looks like resetting the bulb did it, I checked it's ip by pinging, turned it off, no ping, then restarted HA with the custom component (I had removed it), after boot I turned on the bulb, added the bulb to the yaml and it validated!! Another reboot and I can control the bulb :)

Thankx for the quick response!
J
